.readthedocs.yml

```
@@ -1,10 +1,14 @@
 # .readthedocs.yml
 
+version: 2
+
 build:
   image: latest
```

`image` is legacy according to the docs.
https://docs.readthedocs.io/en/stable/config-file/v2.html#legacy-build-specification

On the other hand, the `build` section is apparently in beta.
```suggestion
  os: ubuntu-20.04
```